charts/kamus/values.yaml

```diff
@@ -10,6 +10,8 @@ image:
   version: 0.2.4.0
   repository: soluto
   pullPolicy: IfNotPresent
+pod:
+  annotations:
```

Comment:
I think you need to add empty JSON there `{}`, let's wait and see if the tests are passing

Reply:
Changed